gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/activity/impl/ProcessWorkUnitImpl.java

```diff
@@ -66,13 +70,36 @@ public class ProcessWorkUnitImpl implements ProcessWorkUnit {
 
   @Override
   public int processWorkUnit(WorkUnitClaimCheck wu) {
+    AutomaticTroubleshooter troubleshooter = null;
+    Optional<EventSubmitter> eventSubmitter = Optional.ofNullable(wu.getEventSubmitterContext()).map(EventSubmitterContext::create);
     try (FileSystem fs = Help.loadFileSystemForce(wu)) {
       List<WorkUnit> workUnits = loadFlattenedWorkUnits(wu, fs);
       log.info("(M)WU [{}] - loaded; found {} workUnits", wu.getCorrelator(), workUnits.size());
       JobState jobState = Help.loadJobState(wu, fs);
-      return execute(workUnits, wu, jobState, fs);
+      log.info("Loaded jobState. Instantiating and starting troubleshooter...");
+      troubleshooter = AutomaticTroubleshooterFactory.createForJob(ConfigUtils.propertiesToConfig(jobState.getProperties()));
+      troubleshooter.start();
+      log.info("Finished instantiating troubleshooter. Now executing flow");
```

Comment:
is there really value in `.info` level debugging?  won't exception logging let us know where something went wrong (following line 77's logging)?